README.md

```diff
@@ -11,41 +11,51 @@ using [Syft](https://github.com/anchore/syft).
 
 By default, this action will execute a Syft scan in the workspace directory
 and upload a workflow artifact SBOM in SPDX format. It will also detect
-if being run during a `release` and attach the SBOM
-as a release asset.
+if being run during a [GitHub release](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases)
+and upload the SBOM as a release asset.
 
 ## Example Usage
 
-### Scan a docker image
+### Scan a container image
 
-Use the `image` parameter
+To scan a container image using the docker daemon use the `image` parameter:
 
 ```yaml
 - uses: anchore/sbom-action@main
   with:
     image: example/image_name
 ```
 
+With a container registry:
+
+```yaml
+- uses: anchore/sbom-action@main
+  with:
+    image: ghcr.io/example/image_name:tag
```

Comment:
Just so I understand clearly, is this method requiring the Docker daemon to be available?

Reply:
yes

Reply:
this will be more clear with the changes in the other PR, there is _another_ example for "To connect directly to the registry without using the docker daemon..."